src/main/java/com/endava/soa_5ed/task3_oop/Cat.java

```diff
@@ -0,0 +1,84 @@
+package com.endava.soa_5ed.task3_oop;
+
+public class Cat extends Feline implements Tail {
+    private double size;
+    private String name;
+
+    public Cat(String gender, String furColor, String name) {
+        super(gender, furColor);
+        this.name = name;
+        System.out.println("############ Creating Cat ############");
+    }
+
+    public Cat(String gender, String furColor, double size, String name) {
+        super(gender, furColor);
+        this.size = size;
+        this.name = name;
+        System.out.println("############ Creating Cat ############");
+    }
+
+    public double getSize() {
+        return this.size;
+    }
+
+    public void setSize(double size) {
+        this.size = size;
```

Comment:
Logs may be also used to print information about the behavior of your program at different level. 
For ex.
log.error - if your app is going down and this is a dangerous situation
log.debug - it's not an usual situation - but your app can handle this, but, in future, you may analyze this message to understand why some functionalities arent working as expected
log.info - just a simple information

Reply:
thank you